extension/war/src/main/resources/locale/navigation/portal/administration_en.properties

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (C) 2020 eXo Platform SAS.
```

Comment:
update header

Reply:
you should declare the file into https://github.com/Meeds-io/social/blob/develop/translations.properties#L64 like done for global site, in order to be considered by crowdin